src/DatePicker.jsx

```diff
@@ -0,0 +1,26 @@
+import { DateRangePicker } from '@mui/x-date-pickers-pro';
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { useState } from "react";
+
+const CustomDatePicker = ({ selectedDateRange, onRangeChange }) => {
+  const [dateRange, setDateRange] = useState(selectedDateRange || [null, null]);
```

Comment:
we typically want our state variables to maintain the same variable type so maybe we should go with an empty array? I think empty arrays are now registered as a falsey value in javascript (recent update). So it should work like null  as Rebecca suggested. [null, null] however, because it is an array that technically has a non-zero length would be a truthy value and might cause some issues.